assets/stylesheets/responsive/custom.scss

```diff
@@ -827,6 +827,55 @@ p.ui-dropdown-list-trigger {
   border-top-color: rgba(255,255,255,0.4);
 }
 
+
 .ui-dropdown-list.active .ui-dropdown-list-trigger {
   border: 0;
 }
+
+.icon-standalone {
+  display: block;
+  width: 22px;
+  height: 22px;
+}
+
+.latest-requests {
+  h4 {
+    font-size: 1.125em;
+    color: #9797A4;
```

Comment:
Should apply the greyer font style to the left hand column too:

![screen shot 2015-06-05 at 12 50 35](https://cloud.githubusercontent.com/assets/282788/8005400/a5568e7a-0b81-11e5-9fa0-129f04040b22.png)